frescobaldi_app/appinfo.py

```diff
@@ -41,3 +41,11 @@
 
 # this one is used everywhere in the application
 appname = "Frescobaldi"
+
+def running_version():
+    import app
+    if app.is_git_controlled():
+        return "{}-git".format(version)
```

Comment:
Adding also the abbreviated commit hash can be useful especially in bug reports, but the resulting string would be too long, probably.
What do you think?